src/GeneralStateTestsFiller/stEWASMTests/extCodeCopyFiller.yml

```diff
@@ -67,9 +67,8 @@ extCodeCopy:
         a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
           balance: "99999958297"
         deadbeef00000000000000000000000000000000:
-          # 0x0061736d010000000104 backwards
           storage: {
-            0: '0x0401000000016d736100'
+            0: '0x61736d01000000010400000000000000000000000000000000000000000000'
```

Comment:
Plase do not loose the zero prefixes here: `0x0061736d01000000010400000000000000000000000000000000000000000000`